tests/cli/test_rasa_train.py

```diff
@@ -558,3 +558,18 @@ def test_get_valid_config(parameters):
 def test_get_valid_config_with_non_existing_file():
     with pytest.raises(SystemExit):
         _get_valid_config("non-existing-file.yml", CONFIG_MANDATORY_KEYS)
+
+
+def test_train_nlu_finetune_with_model(
+    run_in_simple_project_with_model: Callable[..., RunResult]
+):
+    temp_dir = os.getcwd()
+
+    assert os.path.exists(os.path.join(temp_dir, "models"))
```

Comment:
Why do we need this assert? In my opinion this tests only if the fixture `run_in_simple_project_with_model` works and that's not the scope of this test case.

Reply:
It is present in another test (that also does not check if the mentioned fixture works) and to me it looked like a good way to have more specific error messages in case the models folder does not exist. However it is true that we are not checking this functionality at this test so I will remove it.